API_RELEASE_NOTES.md

```diff
@@ -1,3 +1,36 @@
+
+> **August 1, 2023 - August 31, 2023**
+
+### Breaking Changes
+* **Deprecation of partner.semanticscholar.org**
+* On December 2023 we will turn off access to partner.semanticscholar.org 
+Ensure you’re using https://api.semanticscholar.org/ before the cutoff date
+
+### New Features
+* **Specter2 embeddings**
```

Comment:
There's a little confusion here. We are serving only the "proximity" adapter from the API. The other models are available on HuggingFace. Also, did you mean to include the "What is it?" "How it works?" prompts? They look like reminders to yourself.

Suggestion:
```
AI2’s SPECTER 2.0 is the latest version of our document embedding models designed specifically for scientific tasks. 
Building upon its predecessor, SPECTER is tailored to generate embeddings for specific tasks, such as Classification, Regression, and Proximity. The API serves pre-computed Proximity embeddings, appropriate for computing the similarity between two documents.  
The model is hosted as [allenai/specter2](https://huggingface.co/allenai/specter2) on HuggingFace, along with the models for other task-specific embeddings.
```

```

Reply:
I did mean to include the "What is it" and "How it works?". These prompts were my reaction or attempt at balancing feedback of "too vague" against my original goal of one line per item brevity. What you suggest here I believe strikes a good balance. 

And thanks for the catch on proximity.